build/distributions/CentOS/7/x86_64/DVD/7-simp_pkglist.txt

```diff
@@ -130,23 +132,22 @@ brltty
 btrfs-progs
 bzip2
 bzip2-libs
-c-ares
-ca-certificates
 cairo
 cairo-gobject
 cal10n
-# caribou* only in rhel-server distribution ISO
 caribou
+# caribou* only in rhel-server distribution ISO
```

Comment:
With the sorting, comments are now below the related packages.  Seems odd.

Reply:
Yeah, that's not great. We should preserve the comments.